dashboard-for-desktop/articles/dashboard-designer/provide-data/supported-data-sources.md

```diff
@@ -16,9 +16,7 @@ The following data source types are supported.
 * [Object Data Source](#object-data-source)
```

Comment:
Line 9 (above): The following data source types are supported.

Please change the period to a colon.